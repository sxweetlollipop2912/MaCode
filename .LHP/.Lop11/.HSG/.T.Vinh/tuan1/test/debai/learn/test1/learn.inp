9
X 2 1000000000
X 5 1000000000
X 9 1000000000
L 2
L 3
X 100 1000000000
X 500 1000000000
L 3
L 4
