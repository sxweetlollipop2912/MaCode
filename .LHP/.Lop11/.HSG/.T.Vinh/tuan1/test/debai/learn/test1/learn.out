3
No
4
5
